src/server/api/routes/members.router.js

```diff
@@ -46,4 +46,56 @@ router.get('/', async (req, res) => {
   return res.json(membersOfBoard);
 });
 
+/**
+ * @swagger
+ * /users/{userId}/boards/{boardId}/members/{memberId}:
+ *  post:
+ *    tags:
+ *    - Members
+ *    summary: add a member to a board
+ *    description:
+ *      Will add a new member to a specific board.
+ *    produces: application/json
+ *    parameters:
+ *     - in: path
+ *       name: userId
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The userId of the owner of the board
+ *     - in: path
+ *       name: boardId
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The boardId of the specific board to get
+ *     - in: path
+ *       name: memberId
+ *       schema:
+ *         type: integer
+ *         required: true
+ *         description: The memberId of the user to add to the board as member
+ *
+ *    responses:
+ *      201:
+ *        description: Candidate added
+ *      5XX:
+ *        description: Unexpected error
+ *      400:
+ *          Invalid Id error
+ *      404:
+ *        description: Invalid entry error
+ */
+
+router.post('/:memberId', async (req, res) => {
+  await membersController.addMember(
+    req.params.userId,
+    req.params.boardId,
+    req.params.memberId,
+  );
+  return res
+    .status(201)
+    .send(`Member ${req.params.memberId} added to board ${req.params.boardId}`);
```

Comment:
We usually return JSON from the API and not a simple string. I think, in this case it would be okay to not return anything as well.

Reply:
removed! Thanks 🙇🏽  🙌🏽